crates/claims/crates/data-integrity/core/src/options.rs

```diff
@@ -13,8 +13,8 @@ pub struct ProofOptions<M, T> {
     pub context: Option<ssi_json_ld::syntax::Context>,
 
     /// Date a creation of the proof.
-    #[serde(default = "xsd_types::DateTime::now")]
-    pub created: xsd_types::DateTime,
+    #[serde(default = "created_default", skip_serializing_if = "Option::is_none")]
```

Comment:
I think we can remove the `serde(default)` attribute. I know it was already there, but I don't think it makes much sense to put a default value if its not there.

Reply:
Yes definitely!